e2e/testcases/notification_test.go

```diff
@@ -702,6 +702,70 @@ func TestNotificationOnSyncStalled(t *testing.T) {
 	}
 }
 
+func TestNotificationOnSyncReconciling(t *testing.T) {
+	rsName := "root-sync-5"
```

Comment:
Just curious, what is the reason for the incrementing number in the name?

Reply:
I think most of these test works very similarly with each other. Giving a unique name for the test data won't affect how the test works, but might reduce the human error factor when working with these.